### PR TITLE
About Page: Convert to an About section overview that links to several new pages within the about APG section

### PR DESCRIPTION
**Pull request by mcking65**
Implements #2423.

[Preview of new About section](https://deploy-preview-207--aria-practices.netlify.app/aria/apg/about/)

___
WAI Preview Link [failed to build](https://github.com/w3c/wai-aria-practices/runs/12526577689?check_suite_focus=true) on 'Update site files' step. _(Last tried on Wed, 05 Apr 2023 02:40:13 GMT)._

### CODE REVIEW

**Comment by mcking65** [comment]
@isaacdurazo 

The about page in this branch provides draft content for the new about page design; it should work well with your new list view. The subpages of the about section, e.g. introduction, will use the design of the practices pages, e.g., [Providing Accessible Names and Descriptions | APG | WAI | W3C](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/).

**Comment by a11ydoer** [comment]
Note: adding new APG pages like list view and supporting url need to be implemented at wai repo so that preview url can be available.

**Comment by mcking65** [comment]
@alflennik  @howard-e 

Now that I have conflicts resolved and all tests passing, how do I trigger a new attempt to build a preview?

**Comment by alflennik** [comment]
Hi @mcking65 the content type referred to here is not HTML, this error is referring to the build system encountering content that doesn't fit a known pattern and therefore cannot be processed. Some content types are "pattern", "example", "practices", "asset" and so on.

Since previously the about page was just a single file, the build script is currently only aware of that one file. With this PR it will become a series of files and the build script will need to be updated to categorize them correctly (and show them).

File: content/about/introduction/introduction.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        <p>
+          The accessibility guidance in the APG is different from accessibility requirements specified by <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> and <a href="https://www.w3.org/TR/wai-aria/">ARIA</a>.
+          WCAG and ARIA are "W3C normative technical standards" while the APG is an "informative" resource.
+          W3C normative technical standards, which are formally known as "W3C Recommendations", have conformance models that define how to comply with the requirements documented in the standard.
+          Those requirements are developed using a rigorous process defined by
+          The <a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>.
+        </p>
+        <p>
+          The APG does not specify normative requirements and thus does not have a conformance model.
+          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href=""https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
```
Comment (nschonni) [inline content/about/introduction/introduction.html:59]: Two double quotes broke the parsing
```suggestion
          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
```

File: content/about/introduction/introduction.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+      <section id="caution-when-deviating">
+        <h2>Exercise Caution If Deviating from APG Guidance</h2>
+        <p>
+          While APG is not a standard, its patterns and practices do convey requirements from normative standards.
+          The patterns and practices describe wide ranges of options that satisfy the requirements of  relevant accessibility specifications, and the examples provide specific implementations that fit the guidance provided by the patterns and practices.
+          It is important to understand, however, that the APG pattern library is not intended to serve as a catalog of all possible, valid, and potentially usable ways of building accessible user interfaces.
+        </p>
+        <p>
+          So, some developers may find it necessary to go beyond  simply combining various APG patterns to creat an experience that supports their UI requirements.
```
Comment (nschonni) [inline content/about/introduction/introduction.html:83]:
```suggestion
          So, some developers may find it necessary to go beyond  simply combining various APG patterns to create an experience that supports their UI requirements.
```

File: content/about/about.html [hunk 3]
```diff
@@ -15,285 +15,51 @@
...
-          <h3 id="c-2-major-contributors-to-version-1-1">Major Contributors to Version 1.1</h3>
+        </li>
+        <li>
+          <h2><a href="introduction/aria-basics.html">ARIA Basics</a></h2>
           <p>
-            While <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices 1.1 is the work of
-            the entire Authoring Practices Task Force and also benefits from many people throughout the open source
-            community who both contribute significant work and provide valuable feedback, special thanks goes to the
-            following people who provided distinctly large portions of the content and code in version 1.1.
+            Ffacilitate success when applying APG guidance by developing a solid understanding of what ARIA is, what accessibility semantics are, and  their purpose and limitations with this section about
```
Comment (nschonni) [inline content/about/about.html:29]:
```suggestion
            Facilitate success when applying APG guidance by developing a solid understanding of what ARIA is, what accessibility semantics are, and  their purpose and limitations with this section about
```

File: content/about/introduction/introduction.html [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        </p>
+        <p>
+          The APG does not specify normative requirements and thus does not have a conformance model.
+          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
+          uses consensus to ensure APG guidance is consistent with all relevant specifications, including ARIA, but its process for doing so is not bound by the W3C Recommendation Track process.
+        </p>
+        <p>
+          Some guidance provided by the APG is not covered by any normative specification.
+          Most notably, the patterns include keyboard interaction models that reflect a combination of one normative requirement specified by ARIA and conventions established by popular operating systems and software.
+          That is, in the section on <a href="#managingfocus">Managing focus</a>, ARIA specifies that the <kbd>Tab</kbd> key should not move focus inside of certain types of widgets.
```
Comment (nschonni) [inline content/about/introduction/introduction.html:65]: The "Managing Focus" link points to an ID `managingfocus` that doesn't exist in the page

File: content/about/Acknowledgements/acknowledgements.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
```
Comment (alflennik) [inline content/about/Acknowledgements/acknowledgements.html:1]: This file is in the "Acknowledgements" category, which is capitalized, but I think the directory shouldn't be capitalized.

File: content/about/introduction/aria-basics.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (alflennik) [inline content/about/introduction/aria-basics.html:1]: My suggestion with this file is that it be given its own folder like the other about pages, that way the content structure is similar across all the about pages as well as across the pattern and practice pages.

**Review by alflennik — CHANGES_REQUESTED** [review]
Hi @mcking65 as I work on integrating this into the website, I have a couple comments about the structure of the content. I like the new additions by the way!

File: content/about/Acknowledgements/acknowledgements.html (hunk 5 above)
Comment (mcking65) [reply content/about/Acknowledgements/acknowledgements.html]: fixed

File: content/about/introduction/aria-basics.html (hunk 6 above)
Comment (mcking65) [reply content/about/introduction/aria-basics.html]: When I put aria-basics.html and introduction.html in the same directory, I was thinking of them both as part of introduction. However, I now see that one is introduction to APG while other is more introduction to ARIA. So, per your suggestion, I separated them.

In the future, there could be additional files or content under aria-basics that is related to the fundamentals of ARIA itself.

**Comment by alflennik** [comment]
The about restructure as it will look on the website can be previewed here: https://deploy-preview-207--aria-practices.netlify.app/aria/apg/about/

**Comment by andreancardona** [comment]
All the content looks good on my end - just deleted some unnecessary spaces! 🚀

File: content/about/acknowledgements/acknowledgements.html [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+          <li>Patrick Lauke</li>
+          <li>Marek Lewandowski</li>
+          <li>Dan Matthew</li>
+          <li>Shane McCarron</li>
+          <li>Victor Meyer</li>
+          <li>Jonathan Neal</li>
+          <li>Philipp Rudloff</li>
+          <li>Joseph Scheuhammer</li>
+          <li>Nick Schonning</li>
+          <li>thomascorthals</li>
```
Comment (andreancardona) [inline content/about/acknowledgements/acknowledgements.html:146]: Is this a user name or full name? `thomascorthals`

**Review by alflennik — APPROVED** [review]
Thank you for addressing my feedback!

**Comment by mcking65** [comment]
@alflennik, my latest commits are not showing up in w3c/wai-aria-practices#207, so the preview is not up-to-date.

**Comment by alflennik** [comment]
@mcking65 I've updated the existing preview link with the latest changes.

**Comment by mcking65** [comment]
@alflennik 

Could you please update the preview again? I just addressed the unresolved feedback and fixed a sentence where I made a rather embarrassing editorial blunder.

**Comment by alflennik** [comment]
@mcking65 okay, it's updated.

**Review by andreancardona — APPROVED** [review]

**Review by a11ydoer — APPROVED** [review]
Contents looks good. I liked the conciseness of role, properties and state in "aria basics" page and the table of "How the APG is organized" in "about page".

**Comment by a11ydoer** [comment]
@mcking65 @alflennik The review is done. Thanks so much for great work.

**Comment by a11ydoer** [comment]
@jnurthen also reviewed this and he said that he did not see any issues.

**Pull request merged**